README.md

```diff
@@ -95,6 +95,14 @@ tach clean
 This will wipe all the configuration generated and enforced by Tach.
 
 
+If you just want to see the external dependencies and external usage of a given file or module in your project, you can use `tach report`.
```

Comment:
Imo `external` is pretty confusing here - when I think external, I think third party dependencies or usages outside of my project, not outside of the boundary. Here's the first result from google - 
<img width="658" alt="image" src="https://github.com/gauge-sh/tach-python-modular-monorepo-framework-tool/assets/5150563/96900601-177e-4370-81a9-54976e75c1d3">
I think simply just saying "dependencies and usages outside of the [boundary/module]" makes sense for here, the argparse docstring, and the output of running the command.